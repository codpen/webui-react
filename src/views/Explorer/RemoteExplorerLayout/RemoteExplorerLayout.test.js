import React from 'react';
import {testStore} from "../../../../Utils";
import {shallow} from "enzyme";
import {Provider} from "react-redux";


const setUp = (props = {}) => {
    const component = shallow(
        <Provider store={testStore()}>
        </Provider>
    );
    return component;
}

describe('Remote Explorer Layout', function () {

    let wrapper;
    beforeEach(() => {
        const props = {};
        wrapper = setUp(props)
    });

    it('should render without crashing', function () {
        expect(wrapper).toHaveLength(1)
    });
});