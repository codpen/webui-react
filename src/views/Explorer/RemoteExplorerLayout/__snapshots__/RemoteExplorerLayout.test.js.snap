// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Remote Explorer Layout renders should match snapshot 1`] = `
<div
  className="d-none d-md-block row"
  data-test="remoteExplorerLayout"
>
  <Col
    className="mb-3"
    lg={12}
    sm={12}
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <span
      className="text-choose-layout"
    >
      Choose Layout: 
        
    </span>
    <Button
      className="ml-2 layout-change-button"
      color="primary"
      onClick={[Function]}
      tag="button"
    >
      <img
        alt="Single Vertical Pane"
        src="single-pane.png"
        style={
          Object {
            "height": 24,
          }
        }
      />
    </Button>
    <Button
      className="ml-2 layout-change-button"
      color="primary"
      onClick={[Function]}
      tag="button"
    >
      <img
        alt="Double Vertical Pane"
        src="double-pane1.png"
        style={
          Object {
            "height": 24,
          }
        }
      />
    </Button>
    <Button
      className="ml-2 layout-change-button"
      color="primary"
      onClick={[Function]}
      tag="button"
    >
      <img
        alt="Triple Vertical Pane"
        src="triple-pane.png"
        style={
          Object {
            "height": 24,
          }
        }
      />
    </Button>
    <Button
      className="ml-2"
      color="success"
      onClick={[Function]}
      tag="button"
    >
      <i
        className="fa fa-arrows"
      />
       Full Screen
    </Button>
  </Col>
</div>
`;
